core/tests/karma.conf.js

```diff
@@ -24,7 +24,7 @@ module.exports = function(config) {
       'third_party/static/angularjs-1.5.8/angular-mocks.js',
       'third_party/static/headroom-js-0.9.4/headroom.min.js',
       'third_party/static/headroom-js-0.9.4/angular.headroom.min.js',
-      'third_party/static/math-expressions-370a77/build/math-expressions.js',
+      'third_party/static/math-expressions-370a77/math-expressions.js',
```

Comment:
But the directory location here still seems the same?

Also btw I think you might need to add me as a code owner too for manifest.json :)

Reply:
Ruh-roh! I forgot to delete my local folder. And yep, I am getting this error running front-end test: 
![screen shot 2018-07-17 at 12 04 51 am](https://user-images.githubusercontent.com/11153258/42795962-0f7f5832-8955-11e8-87e9-3cbb0201ae81.png)

Reply:
OK, that's not a good sign. What if you just move it one commit ahead?

Worst case maybe we use 8 chars as the hash prefix or something.

Reply:
Looks like ever since this commit : https://github.com/kisonecat/math-expressions/commit/9b7a1d94329a6c73b7db6d513824be609fb1932f
. which is the last commit for v1.7.0, math-expressions.js is no longer compatible with us :(

Reply:
Is that something that is resolvable though? The README seems to indicate that it can be used standalone: https://github.com/kisonecat/math-expressions#code-example

Reply:
For every release since v2.0.0 , there was a different error: It ranges from `SyntaxError: Unexpected token import ` to   `Uncaught ReferenceError: global is not defined`. I can roll it up to v.1.7.0 currently, which is 5 commits after our last commit point of 370a77

Reply:
Here is the latest issue filed over there : https://github.com/kisonecat/math-expressions/issues/13

Reply:
Ah ok, sounds good. Yes, let's definitely use a stable version.

I assume you've tested 1.7.0 manually and it works fine?